### PR TITLE
Add LCM function

### PR DESCRIPTION
**Pull request by lfidelino**
For example,
An input of 12 and 5 would return 60.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "babel-preset-stage-2": "6.13.0",
```
Comment (kentcdodds) [inline package.json]: Thanks! But please avoid upgrading packages unless it's absolutely necessary. In this case, I don't want these packages upgraded. Could you revert these changes please?

**Comment by lfidelino** [comment]
I kinda did the last commit wrong. Thanks for the help, though!
